src/ol/tile.js

```diff
@@ -4,6 +4,7 @@ goog.require('ol');
 goog.require('ol.events.EventTarget');
 goog.require('ol.events.EventType');
 
+var lastSequenceNumber = 0;
```

Comment:
Actually, I just realized in this latest version of the patch I had simplified it enough that the sequence number was only ever explicitely referenced in the assert statement checking that the order of the tiles in the list was correct. Since this will now be tested in a unit test, we can get rid of the sequence number/uid stuff altogether, further simplifying things.